src/modules/api/role/dtos/index.ts

```diff
@@ -1,7 +1,15 @@
-import { IsOptional, IsString } from "class-validator";
+import { IsArray, IsOptional, IsString } from "class-validator";
 
 export class FetchRolesDto {
     @IsOptional()
     @IsString()
     roleName: string;
 }
+
+export class CreateRoleDto {
+    @IsString()
+    roleName: string;
+
+    @IsArray()
```

Comment:
Your validation must allow an array of integer numbers. Use strict validation here